go.mod

```diff
@@ -251,9 +251,9 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.14.11 => github.com/ethereum-optimism/op-geth v1.101408.1-0.20241002211323-d5a96613c22b
+replace github.com/ethereum/go-ethereum v1.14.11 => github.com/roberto-bayardo/op-geth v0.0.0-20241016220820-6c32375dda12
```

Comment:
We should merge the op-geth PR first and then update this

Reply:
right, had this in there to make sure the CI could pass.

Reply:
@roberto-bayardo I merged the `op-geth` PR to unblock this, just noting https://github.com/ethereum-optimism/op-geth/pull/402#pullrequestreview-2372229757 for follow up